src/sentry/api/endpoints/organization_artifactbundle_assemble.py

```diff
@@ -16,8 +17,25 @@
 from sentry.utils import json
 
 
+class OrganizationArtifactBundleAssembleMixin:
+    @classmethod
+    def find_missing_chunks(cls, organization, chunks):
+        """
+        Returns a list of chunks which are missing for an org.
+        """
+        owned = set(
+            FileBlobOwner.objects.filter(
+                blob__checksum__in=chunks, organization_id=organization.id
+            ).values_list("blob__checksum", flat=True)
+        )
+
+        return list(set(chunks) - owned)
```

Comment:
Out of curiosity: did we handle case insensitivity in the other place?

Reply:
I did copy the impl from the debug files which is case sensitive but I would assume chunks are case sensitive.